### ISSUE TITLE
"flakeheaven plugins" fails due to pylint

### ISSUE THREAD

**Issue by mcarans** [flakeheaven/flakeheaven#4]
The command line: `flakeheaven plugins` fails with:

"pylint plugin reports no version" in
  File "lib/python3.8/site-packages/flakeheaven/commands/_plugins.py", line 17, in <genexpr>
  version_width = max(8, max(len(p['version']) for p in plugins))

**Comment by mcarans** [comment]
This is tricky to debug and I have to admit that I don't fully understand the code. 

I had added https://github.com/flakeheaven/flakeheaven/blob/master/flakeheaven/_logic/_discover.py#L50-L68 to fix an existing error to do with pylint not being found

However, having looked now, it appears like the pylint plugin is found twice as "flake8-pylint-plugin" and "pylint". I have experimented and found I can remove the extra discovery code, but only if I remove the pylitn specific plugin code in flakeheaven. See a branch I have made to play aroudn with this: https://github.com/mcarans/flakeheaven/tree/pylint_fix
